src/pages/Login/Login.js

```diff
@@ -1,7 +1,100 @@
+import { useState } from "react"
+import { useAuth } from "../../Context/AuthContext/AuthContext"
+import { Link } from "react-router-dom";
 export default function Login(){
+
+    const {handleLoginData} = useAuth();
+
+    const [login, setLogin] = useState({
+        email:'',
+        password:''
+    }) 
+    const [isError, setIsError] = useState('')
+    const handleData = (e)=>{
+        setLogin({...login, [e.target.name]:e.target.value})
+    }
+    console.log(login)
+
+
+    const loginHandler = (e) =>{
+        const {email, password} = login
+        e.preventDefault();
+        if(email!== '' && password!==''){
```

Comment:
instead of email !== '' you can just use "!email" It checks for any falsy value.

Reply:
sure i will try that.